README.md

```diff
@@ -22,7 +22,10 @@ Sample Set #1                    |  Sample Set #2
 - Select Pokemon by name or by index number
 - Ability to change the Desktop Wallpaper & the Terminal background
 - Internal search system for finding Pokemon
-- Supports iTerm2, Terminology & Tilix
+- Supports iTerm2, Terminology, Tilix, Terminator<sup>1</sup> and XFCE4Terminal <sup>2</sup>
```

Comment:
Bro, did you actually got it? The commit on your fork was supposed to show up here, you need to push the commit in your fork so we see it.